src/utils/url-utils.ts

```diff
@@ -0,0 +1,102 @@
+import { useHistory } from 'react-router-dom';
+import moment from 'moment';
+
+export type URLParams = {
+  hazardLayerId?: string;
+  baselineLayerId?: string;
+  date?: number;
+};
+
+/*
+  transforms the string value according to the field specified in the
+  type URLParams
```

Comment:
This function and the one below are clearly doing something with dates but they avoid the topic entirely. Maybe give them better names and docs like `parseIfDate` and consider combining the two to one function with a boolean to change between valueOf and format